README.md

```diff
@@ -35,112 +46,107 @@ $ cd ios
-
-* `numberVariation(evalutionId: string, defaultValue?:number)`
 
-* `jsonVariation(evalutionId: string, defaultValue: any)`
-
-* `registerListener(listener: (type: string, flags: any) => void) `
-
-* `unregisterListener(listener: (type: string, flags: any) => void)`
-
-* `destroy()`
-<br><br>
-
-
-## Fetch evaluation's value
-It is possible to fetch a value for a given evaluation. Evaluation is performed based on different type. In case there is no evaluation with provided id, the default value is returned.
-
-Use appropriate method to fetch the desired Evaluation of a certain type.
-### <u>_boolVariation(evaluationId: string, defaultValue?: boolean)_</u>
-
-```JavaScript
-//get boolean evaluation
-let evaluation = await client.boolVariation("demo_bool_evaluation", false)
-```
-### <u>_numberVariation(evaluationId: string, defaultValue?:number)_</u>
-```JavaScript
-//get number evaluation
-let numberEvaluation = await client.numberVariation("demo_number_evaluation", 0)
+Using Expo, initialize a project. For the best experience, select `Minimal` as the template when prompted.
+```shell
+expo init SampleProject
+cd SampleProject
 ```
 
-### <u>_stringVariation(evaluationId: string, defaultValue?:string)_</u>
-```JavaScript
-//get string evaluaation
-let stringEvaluation = await client.stringVariation("demo_string_evaluation", "default");
+Add the Harness React Native SDK to the `package.json` under the `dependencies` section.
+```json
+"@harnessio/ff-react-native-client-sdk": "^1.0.1"
 ```
-### <u>_jsonVariation(evaluationId: string, defaultValue?: any)_</u>
-```JavaScript
-//get json evaluation
-let jsonEvaluation = await client.jsonVariation("demo_json_evaluation", {});
 
-```
+Replace the code in `App.js` with the following Sample Code, and replace `apiKey` with your Client SDK Key.
+```javascript
+import cfClientInstance, {CfConfiguration, CfTarget} from '@harnessio/ff-react-native-client-sdk';
 
-## _Register for events_
-This method provides a way to register a listener for different events that might be triggered by SDK, indicating specific change in SDK itself.
+export default function App() {
+  const flagName = 'harnessappdemodarkmode';
+
+  const [client, setClient] = useState(null);
+  const [flagValue, setFlagValue] = useState(null);
+
+  async function initializeClient() {
+    let cfClient = cfClientInstance;
+    let cfConfig = new CfConfiguration();
+    cfConfig.streamEnabled = true;
+
+    const cfTarget = new CfTarget();
+    cfTarget.identifier = 'Harness RN Sample App'
+
+    const apiKey = "your-client-sdk-key";
+
+    try {
+      await cfClientInstance.initialize(apiKey, cfConfig, cfTarget);
+    } catch (err) {
+      console.log(err);
+    }
+    setClient(cfClient);
+  }
+
+  async function evalFlag() {
+    let res = await client.boolVariation(flagName, false);
+    setFlagValue(res.value);
+  }
+
+  useEffect(() => {
+    if (client == null) {
+      initializeClient();
+    } else {
+      evalFlag();
+    }
+  });
```

Comment:
this code will render on component mount and on any state change.

Reply:
Yeah, for React-Native, I wanted to just bring up a simple Simulator that showed the flag and it's current value and test whether the Client and connection work. I didn't want to over complicate the sample with stream, polling, or metrics with this snippet. I could possibly point to the Sample repo for a more complete sample application. If there is an easier way to accomplish all of that, I'd definitely take the suggestion.